builds/misc/packages-release.yaml

```diff
@@ -435,5 +435,161 @@ stages:
+            then
+                sudo apt-get install jq -y
+            fi
+
+      - task: Bash@3
+        displayName: Get Parameters From Azure-iotedge
+        condition: true
+        name: GitHubParameters
+        inputs:
+          targetType: 'inline'
+          script: |
+            # Source the scripts and get versions
+            edgeletVersion=$(cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-edge.json | jq ".\"aziot-edge\"" | tr -d '"')
+            iisVersion=$(cat cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-identity-service.json | jq ".\"aziot-identity-service\"" | tr -d '"')
+
+            echo "##vso[task.setvariable variable=edgeletVersion;isOutput=true]$edgeletVersion"
+            echo "##vso[task.setvariable variable=iisVersion;isOutput=true]$iisVersion"
+        env:
+          GITHUB_PAT: "$(GitHubAccessToken)"
+      - task: Bash@3
+        displayName: Download Edgelet GitHub
+        condition: true
+        inputs:
+          targetType: 'inline'
+          script: |
+            edgeletVersion=$(GitHubParameters.edgeletVersion)
+            iisVersion=$(GitHubParameters.iisVersion)
+
+            wget https://github.com/Azure/azure-iotedge/releases/download/$edgeletVersion/aziot-edge_$edgeletVersion-1_$(os)_$(arch).$(ext) -O $(System.ArtifactsDirectory)/aziot-edge_$edgeletVersion-1_$(os)_$(arch)_github.$(ext)
+            wget https://github.com/Azure/azure-iotedge/releases/download/$edgeletVersion/aziot-identity-service_$iisVersion-1_$(os)_$(arch).$(ext) -O $(System.ArtifactsDirectory)/aziot-identity-service_$iisVersion-1_$(os)_$(arch)_github.$(ext)
+
+      - task: Bash@3
+        displayName: Released Artifacts Smoke Tests
+        condition: true
+        inputs:
+          targetType: 'inline'
+          script: |
+            # Source the scripts & Update version files
+            sudo chmod +x $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+            source $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+
+            if [[ "$(os)" == "ubuntu"* ]]; then
+            # Ubuntu
+              if [[ "$(os)" == *"20.04" ]]; then
+                setup-focal-source-apt
+                setup-config-apt "https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb"
+              elif [[ "$(os)" == *"18.04" ]]; then
+                setup-config-apt "https://packages.microsoft.com/config/ubuntu/18.04/multiarch/packages-microsoft-prod.deb"
+              else
+                echo "Unsupported OS: $(os)"
+                exit 1;
+              fi
+            elif [[ "$(os)" == "debian11" ]]; then
+              # Debian
+              echo "Setup source artifact repository"
+              curl https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb > ./packages-microsoft-prod.deb
+              sudo apt install ./packages-microsoft-prod.deb
+              sudo apt-get update
```

Comment:
Shouldn't this logic have been part of `setup-config-apt` in smokeTestHelper.sh?

Reply:
Done.